packages/components/src/internal/renderers/AncestorRenderer.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2019 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { Map } from 'immutable';
+import { DefaultRenderer } from './DefaultRenderer';
+
+interface AncestorRendererProps {
+    data: Map<any, any>;
+}
+
+export const ANCESTOR_LOOKUP_CONCEPT_URI = 'http://www.labkey.org/types#ancestorLookup';
```

Comment:
Should this be moved with the other Concept_Uri constants? `src/internal/components/domainproperties/constants.ts`